### PR TITLE
Adds Operative EquipSet

### PR DESCRIPTION
**Pull request by SconesC**
Adds an Operative admin-usable Equipset. How we've gone this long without it is beyond me.

### CODE REVIEW

**Comment by Xhuis** [comment]
f e a t u r e

File: code/modules/admin/verbs/debug.dm [hunk 1]
```diff
@@ -640,6 +640,7 @@ var/global/list/g_fancy_list_of_types = null
 		"red wizard",
 		"marisa wizard",
 		"plasmaman"
```
Comment (Jordie0608) [inline code/modules/admin/verbs/debug.dm]: Need a comma here

File: code/modules/admin/verbs/debug.dm [hunk 2]
```diff
@@ -1000,8 +1001,16 @@ var/global/list/g_fancy_list_of_types = null
...
 			M.equip_to_slot_or_del(new /obj/item/weapon/tank/internals/plasmaman/full(M),slot_back)
 			M.equip_to_slot_or_del(new /obj/item/clothing/mask/breath(M),slot_wear_mask)
-
-
+			
+		if("operative")
+			M.equip_to_slot_or_del(new /obj/item/clothing/under/syndicate(M), slot_w_uniform)
+			M.equip_to_slot_or_del(new /obj/item/clothing/shoes/jackboots(M), slot_shoes)
+			M.equip_to_slot_or_del(new /obj/item/clothing/gloves/combat(M), slot_gloves)
+			M.equip_to_slot_or_del(new /obj/item/weapon/storage/belt/military(M), slot_belt)
```
Comment (Supermichael777) [inline code/modules/admin/verbs/debug.dm]: don't nukeops start with a steichkn here? or are these special sauce ops?

**Comment by Dorsisdwarf** [comment]
Can you add "Tactical operative" which is Operative but already kitted out fully,  with a shotgun and a backpack full of shotgun ammo?

**Pull request closed without merging**